fdbserver/workloads/RemoveServersSafely.actor.cpp

```diff
@@ -285,6 +301,8 @@ struct RemoveServersSafelyWorkload : TestWorkload {
 		return killableProcesses;
 	}
 
+	// toKill1 and toKill2 are a random subset of all processes. If simply kill all processes in toKill1 or toKill2,
```

Comment:
```suggestion
	// toKill1 and toKill2 are two random subsets of all processes. If simply kill all processes in toKill1 or toKill2,
```

Reply:
I don't know much details about the process killing part but is it possible to add something like `totalNumProcessesToKill` to this workload so that it won't kill too many processes?

Reply:
the number is controlled by the test workload .txt file. It is hard to assign a static number to it because each simulation has different number of processes.

Reply:
What about a ratio of the existing processes? Like 1% or something.